routes/web.php

```diff
@@ -22,6 +22,16 @@
     Route::resource('locations', 'LocationsController', [
         'parameters' => ['location' => 'location_id']
     ]);
+    
+    Route::get(
+        '{locationId}/printonlyassignedtolocation',
```

Comment:
This isn't important, just a stylistic thing, but these route names/URLs are super long. Maybe more like `print-assigned`?

Reply:
done